src/super_gradients/training/utils/callbacks/callbacks.py

```diff
@@ -1070,7 +1069,7 @@ class ExtremeBatchCaseVisualizationCallback(Callback, ABC):
 
     :param freq: int, epoch frequency to perform all of the above (default=1).
 
-     Inheritors should implement process_extreme_batch which returns an image, as an np.array (uint8) with shape BCHW.
+     Inheritors should implement process_extreme_batch which returns an image, as np.array (uint8) with shape BCHW.
```

Comment:
This should be updated if we do decide to change to channels last shouldn't it ?

Reply:
Fixed in 08c643f19ffa35147b60787aa46db9269076b163